frontend/src/components/adminmessage.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+<span>
+    <div v-if="(this.message !== undefined && this.message.Display === true) && !hide" id ="messagebox">
+        <div style="width: 100%;float:left;" v-html="this.message.Message"></div>
+        <div @click="hide = !hide" style="cursor: pointer;position:absolute;right:10px;top:6px;color:#333;font-size:20px;">&times;</div>
+    </div>
+</span>
+</template>
+<script lang="ts">
+// This component handles rendering of a single administrator message
+
+import Vue from 'vue';
+import AdminMessageUpdate from '@/structures/adminMessageUpdate';
```

Comment:
I don't think the admin messages are updated properly... Everything looks fine with the inspector in the admin panel (all the statuses are 200), but I don't see a message pop up in the shuttle tracker when I submit one